### DISCUSSION TITLE
Use jenkins-plugins role with a Jenkins docker container

### DISCUSSION THREAD

**Discussion by Jordi-Pineiro** [wcm-io-devops/ansible-jenkins-plugins#14] [Show and tell]
Hi,

I would like to open a PR with a change allowing to change the default values of  owner and group parameters in community.general.jenkins_plugin.

This way the role can be used when Jenkins is running as a Docker container.

Could you please grant me access to create a branch and open the PR? 

Thanks!!!

**Comment by tobias-richter** [comment]
Hi @Jordi-Pineiro thank you for your contribution. I propose to fork the repository and do the changes in the fork. You can then create a pull request from there.

**Comment by Jordi-Pineiro** [comment]
Hi @tobias-richter, done as you told me.

**Comment by tobias-richter** [comment]
@Jordi-Pineiro PR #15 was merged and I have release version 1.7.0 of the role. Thank you for your contribution.
